### ISSUE TITLE
Recommend improvements to the "Run a Validator" documentation

### ISSUE THREAD

**Issue by kenny-io** [shardeum/shardeum-docs#314]
- Follow the steps documented on https://docs.shardeum.org/node/run/validator
- Note down any issues or differences in the documented experience
- Update the docs to reflect and account for your noted experiences
- Send a PR

**Comment by kaksv** [comment]
Is this still an issue so that I take it on @kenny-io

**Comment by kenny-io** [comment]
Hi @kaksv yes this issue is not assigned yet and thank you for your interest. However you have another issue in progress [here](https://github.com/shardeum/network-status/pull/5) let's wrap that one up before starting a new one.

**Comment by kaksv** [comment]
Thank you. Let me have a look at it
